### PR TITLE
Add designview to the release notes

### PR DESCRIPTION
**Pull request by hevayo**

### CODE REVIEW

File: docs/release-notes/3.3.0-release-note.md [hunk 1]
```diff
@@ -1,9 +1,10 @@
 # Ballerina VSCode Plugin (Version 3.3.0) 
 
 We are happy to announce the Ballerina VSCode plugin 3.3.0 release, which has some exciting new features. Following are the highlights of this release.
-- Visual Data Mapper - Helps you write and visualize data transformations easily
-- GraphQL Tryit - Facilitates  trying out the GraphQL services with the integrated client 
-- Improved record editor - Provides a better editing experience with suggestions 
+- [Visual Data Mapper](#data-mapper) - Helps you write and visualize data transformations easily
+- [GraphQL Tryit](#integrated-graphql-tryit) - Facilitates  trying out the GraphQL services with the integrated client
```
Comment (praneesha) [inline docs/release-notes/3.3.0-release-note.md:5]: Seems like there is an extra space after the word `trying` here.

File: docs/release-notes/3.3.0-release-note.md [hunk 2]
```diff
@@ -1,9 +1,10 @@
 # Ballerina VSCode Plugin (Version 3.3.0) 
 
 We are happy to announce the Ballerina VSCode plugin 3.3.0 release, which has some exciting new features. Following are the highlights of this release.
-- Visual Data Mapper - Helps you write and visualize data transformations easily
-- GraphQL Tryit - Facilitates  trying out the GraphQL services with the integrated client 
-- Improved record editor - Provides a better editing experience with suggestions 
+- [Visual Data Mapper](#data-mapper) - Helps you write and visualize data transformations easily
+- [GraphQL Tryit](#integrated-graphql-tryit) - Facilitates  trying out the GraphQL services with the integrated client 
+- [Improved record editor](#record-editor) - Provides a better editing experience with suggestions 
+- [Project Design View](#record-editor) - Visualize service intaractions in your project
```
Comment (praneesha) [inline docs/release-notes/3.3.0-release-note.md:7]: Let's say `Facilitates visualizing service interactions in your project` to preserve consistency with the other points.

There is a typo in the word `interactions`.

File: docs/release-notes/3.3.0-release-note.md [hunk 3]
```diff
@@ -26,5 +27,12 @@ If you are new to Ballerina, you can download the [installers](/downloads/#swanl
 - Now, you can create records by importing a JSON file. All you need to do is select a sample JSON file under the `Import JSON file` section in the record creation.
 - Record creation using JSON samples now supports separate record creation for complex JSON records. 
 
-Apart from the above, there are lots of other bug fixes and improvements. For more details, see the [issue list](). 
+## Project Design View (Experimental)
+![design-view](./../../resources/release-notes/3.3.0/design-view.gif)
+- This release include an experimental feature that allows you to visualize service interactions in your project
```
Comment (praneesha) [inline docs/release-notes/3.3.0-release-note.md:32]: Should be `This release includes....`.

File: docs/release-notes/3.3.0-release-note.md [hunk 4]
```diff
@@ -26,5 +27,12 @@ If you are new to Ballerina, you can download the [installers](/downloads/#swanl
...
+- It also comes with a view to see record type compostions and relationships
```
Comment (praneesha) [inline docs/release-notes/3.3.0-release-note.md:33]: Should be `view to see record-type compositions and relationships`. A hyphen is missing in `record-type` and there is a typo in the word `compositions`.

File: docs/release-notes/3.3.0-release-note.md [hunk 5]
```diff
@@ -26,5 +27,12 @@ If you are new to Ballerina, you can download the [installers](/downloads/#swanl
 - Now, you can create records by importing a JSON file. All you need to do is select a sample JSON file under the `Import JSON file` section in the record creation.
 - Record creation using JSON samples now supports separate record creation for complex JSON records. 
 
-Apart from the above, there are lots of other bug fixes and improvements. For more details, see the [issue list](). 
+## Project Design View (Experimental)
+![design-view](./../../resources/release-notes/3.3.0/design-view.gif)
+- This release include an experimental feature that allows you to visualize service interactions in your project
+- It also comes with a view to see record type compostions and relationships 
+- Use command "Ballerina: Project Design" to open design view
```
Comment (praneesha) [inline docs/release-notes/3.3.0-release-note.md:34]: Should be `Use the `Ballerina: Project Design` command to open the design view.`

**Review by keizer619 — APPROVED** [review]

**Pull request merged**